python/ray/tune/trainable.py

```diff
@@ -330,6 +330,15 @@ def restore(self, checkpoint_path):
         self._timesteps_since_restore = 0
         self._iterations_since_restore = 0
         self._restored = True
+        logger.info("Restored from checkpoint: {}".format(checkpoint_path))
+        state = {
+            "iter": self._iteration,
+            "ts": self._timesteps_total,
+            "time": self._time_total,
+            "eps": self._episodes_total
+        }
+        logger.info("Current state after restoring:" + ", ".join(
```

Comment:
Might be cleaner to just print the dict
```python
logger.info("Current state after restoring: {}".format(state))
```

The output would look something like:
```
Current state after restoring: {'iter': 1, 'ts': 100, 'time': 1000, 'eps': 1e-06}
```

Instead of 
```
Current state after restoring:iter: 1, ts: 100, time: 1000, eps: 1e-06
```